### PR TITLE
Add TTN Mapper template

### LINKED ISSUE #14: TTN Mapper
#### Summary

Add a webhook template to forward gps tracker data to TTN Mapper.

#### Why do we need this?

Map coverage of TTNv3, and possibly also private instances.

#### Environment

The Things Stack v3.10.5 and v3.11.0-rc1

#### How do you propose to implement this?

Write a template that contains all the relevant info.

#### Can you do this yourself and submit a Pull Request?

Yes, but there are some data that is required to identify the network, which is currently not available in the webhook. See https://github.com/TheThingsNetwork/lorawan-stack/issues/3705

### PR DESCRIPTION
**Pull request by jpmeijers**
#### Summary

Closes #14 

Depends on #20, #17 and #18 to be solved, but for now it's still usable, albeit with a smaller feature set.

#### Changes

- Add template for TTN Mapper
- Add reference to templates list

#### Notes for Reviewers

Commented out code which adds a field for the experiment ID is there until #17 can be solved. At that point we can enable those lines again.

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Compatibility: The changes are backwards compatible, they don't break existing deployments.
- [X] Testing: The changes are tested. -- tested against a private hosted v3.11.2 stack.
- [X] Documentation: Relevant documentation is added or updated.

### CODE REVIEW

**Review by nejraselimovic — APPROVED** [review]

**Comment by nejraselimovic** [comment]
@jpmeijers is there any documentation about integrating TTNMapper with V3? If not, could you create something I could use for [our documentation page](https://thethingsindustries.com/docs/)?

**Comment by jpmeijers** [comment]
I'll be hosting the documentation on TTN Mapper's side, as defined in the template: https://ttnmapper.org/docs/tts/v3/

The documentation is currently only a placeholder, but I plan to write it as soon as I know if the webhook template is accepted in its current form or not.

**Comment by jpmeijers** [comment]
Old documentation on which we can base the new docs:
https://www.thethingsnetwork.org/docs/applications/ttnmapper/index.html

File: ttnmapper.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+template-id: ttnmapper
+name: TTN Mapper
+description: Contribute data for coverage and signal analysis.
```
Comment (nejraselimovic) [inline ttnmapper.yml:3]:
```suggestion
description: Contribute data for coverage and signal analysis
```

**Review by nejraselimovic — APPROVED** [review]

**Comment by nejraselimovic** [comment]
> Old documentation on which we can base the new docs:
> https://www.thethingsnetwork.org/docs/applications/ttnmapper/index.html

Thank you for the info. This template looks fine to me, just remove the dot to keep it consistent with other templates, and let's keep going with the documentation and fixes you proposed.

**Comment by nejraselimovic** [comment]
> > Old documentation on which we can base the new docs:
> > https://www.thethingsnetwork.org/docs/applications/ttnmapper/index.html
> 
> Thank you for the info. This template looks fine to me, just remove the dot in description to keep it consistent with other templates, and let's keep going with the documentation and fixes you proposed.

**Pull request merged**